pkg/core/resources/store/store.go

```diff
@@ -46,9 +46,6 @@ func (s *strictResourceStore) Create(ctx context.Context, r model.Resource, fs .
 	if opts.Name == "" {
```

Comment:
And also we could place checks that Mesh is always specified if a resource has ScopeMesh

Reply:
Good point!